evap/settings.py

```diff
@@ -36,6 +36,7 @@
 VOTER_COUNT_NEEDED_FOR_PUBLISHING_RATING_RESULTS = 2
 VOTER_PERCENTAGE_NEEDED_FOR_PUBLISHING_AVERAGE_GRADE = 0.2
 SMALL_COURSE_SIZE = 5  # up to which number of participants the evaluation gets additional warnings about anonymity
+PARTICIPATION_DELETION_AFTER_INACTIVE_MONTHS = 18 * 30  # unit in days, i.e. 18 months = 18 * 30
```

Comment:
`PARTICIPATION_DELETION_AFTER_INACTIVE_MONTHS` heavily implies that the unit is "months" -- if we expect any other unit in code, we will shoot ourselved in the foot with that sooner or later.

Was there a specific reason to use "days" here? If not, I'd just use months. I don't think we will need sub-month precision. Using the "30 days per month" approximation is probably valid enough in the code that needs to create timedeltas in days.

Reply:
Is there any reason to not specify a timedelta directly here in the settings?

Reply:
We can do that. I was hesitant to suggest it since we haven't done that for any setting yet. I was thinking of migrating to `datetime` and `fraction` instances in a separate PR. We have used `Fraction`s for one setting and it didn't cause any problems, so introducing `datetime` here would probably be fine.

Reply:
> Was there a specific reason to use "days" here? If not, I'd just use months. I don't think we will need sub-month precision. Using the "30 days per month" approximation is probably valid enough in the code that needs to create timedeltas in days.

The biggest unit in timedelta is weeks, there is no option for months. But I thought days make more sense, i.e. 30 days are more accurate than 4 weeks. There are parameters for days, seconds, microseconds, milliseconds, minutes, hours and weeks.

But I can also just use the timedelta here, which makes it probably also less confusing.

Reply:
I think that using timedelta directly is better than the current version; I think I prefer it over having "30 * days" hardcoded - managers can still choose to use this calculation, but could also put something like `days = 2 * 365`. @richardebeling ?